### DISCUSSION TITLE
Custom HTTP API

### DISCUSSION THREAD

**Discussion by lple994** [blakeblackshear/frigate#9508] [Ask A Question]
Thank you to those that have developed Frigate NVR,

I want to create custom HTTP GET requests for Frigate to return information about zones, detection, etc. I am using the docker image from "ghcr.io/blakeblackshear/frigate:stable". 
I am aware that the provided MQTT messaging setup is more comprehensive. But, I don't want to run an additional sever.

Should I clone the repository and modify the source code? How would I build a docker image after the modification? 

Any advice is appreciated

**Comment by bogdanr** [comment]
The Dockerfile is here:
https://github.com/blakeblackshear/frigate/blob/dev/docker/main/Dockerfile
You could use this to build your custom container after you change the desired files.
